backend/src/test/java/de/neuefische/backend/service/WizardServiceTest.java

```diff
@@ -35,26 +33,31 @@ void listOfWizardExpectEmptyList() {
     @Test
     void findById() throws IllegalAccessException {
 
+        //GIVEN
+        Wizard givenWizard= new Wizard("testId","aaa","aaa","sss","bbb","ddd");
```

Comment:
You might as well use "real" attributes instead of "aaa" :stuck_out_tongue_winking_eye: